paypal_checkout.php

```diff
@@ -264,21 +264,33 @@ public function validate(array $get, array $post)
         $payload = file_get_contents('php://input');
         $webhook = json_decode($payload);
 
+        // Discard all webhook events, except when the order is completed or approved
+        $events = ['CHECKOUT.ORDER.COMPLETED', 'CHECKOUT.ORDER.APPROVED'];
```

Comment:
Have you been able to trigger this 'CHECKOUT.ORDER.APPROVED' webhook? The only ones triggered in my testing are 'CHECKOUT.ORDER.COMPLETED' and 'PAYMENT.CAPTURE.COMPLETED'.  I think catching 'PAYMENT.CAPTURE.COMPLETED' instead of 'CHECKOUT.ORDER.COMPLETED' would be sufficient unless you see a problem with it.  

$webhook->resource->purchase_units[0] wouldn't exist on this webhook so we'd need to update the code below.